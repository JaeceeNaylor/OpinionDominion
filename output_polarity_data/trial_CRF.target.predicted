SentenceID: 1016296:2
Polarity: negative

SentenceID: 1016296:3
Polarity: positive

SentenceID: 1016296:4
Polarity: positive

SentenceID: 1016296:4
Polarity: positive

SentenceID: 1016296:4
Polarity: positive

SentenceID: 1016296:5
Polarity: positive

SentenceID: Z#3:0
Polarity: positive

SentenceID: Z#3:0
Polarity: positive

SentenceID: Z#3:1
Polarity: positive

SentenceID: Z#3:2
Polarity: positive

SentenceID: Z#3:2
Polarity: positive

SentenceID: Z#3:3
Polarity: positive

SentenceID: PP#9:0
Polarity: positive

SentenceID: PP#9:1
Polarity: positive

SentenceID: PP#9:2
Polarity: positive

SentenceID: PP#9:2
Polarity: positive

SentenceID: PP#9:3
Polarity: positive

