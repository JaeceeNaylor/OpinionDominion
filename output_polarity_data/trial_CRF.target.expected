SentenceID: Y#9:4
Polarity: positive

SentenceID: Y#9:4
Polarity: positive

SentenceID: Y#9:5
Polarity: positive

SentenceID: Z#5:0
Polarity: positive

SentenceID: Z#5:1
Polarity: negative

SentenceID: Z#5:2
Polarity: positive

SentenceID: Z#5:3
Polarity: positive

SentenceID: Z#5:4
Polarity: negative

SentenceID: Z#5:4
Polarity: negative

SentenceID: Z#5:5
Polarity: negative

SentenceID: Z#5:6
Polarity: positive

SentenceID: Z#5:6
Polarity: negative

SentenceID: TM#4:0
Polarity: positive

SentenceID: TM#4:1
Polarity: neutral

SentenceID: TM#4:1
Polarity: positive

SentenceID: TM#4:2
Polarity: positive

SentenceID: TM#4:2
Polarity: negative

SentenceID: TM#4:2
Polarity: negative

SentenceID: TM#4:3
Polarity: negative

SentenceID: TM#4:4
Polarity: positive

SentenceID: TFS#11:0
Polarity: negative

SentenceID: TFS#11:1
Polarity: negative

SentenceID: TFS#11:3
Polarity: negative

SentenceID: TFS#11:3
Polarity: negative

SentenceID: TFS#11:3
Polarity: negative

SentenceID: TFS#11:4
Polarity: negative

SentenceID: TFS#11:5
Polarity: negative

SentenceID: TFS#11:6
Polarity: negative

SentenceID: ADLT#10:0
Polarity: positive

SentenceID: ADLT#10:1
Polarity: positive

SentenceID: ADLT#10:2
Polarity: positive

SentenceID: ADLT#10:3
Polarity: positive

SentenceID: ADLT#10:3
Polarity: positive

SentenceID: ADLT#10:3
Polarity: positive

SentenceID: ADLT#10:5
Polarity: positive

SentenceID: ADLT#10:6
Polarity: positive

SentenceID: ADLT#10:6
Polarity: positive

SentenceID: ADLT#10:6
Polarity: positive

SentenceID: ADLT#10:6
Polarity: positive

SentenceID: ADLT#10:7
Polarity: positive

SentenceID: PP#3:0
Polarity: positive

SentenceID: PP#3:1
Polarity: positive

SentenceID: PP#3:3
Polarity: positive

SentenceID: BHD#6:0
Polarity: positive

SentenceID: BHD#6:1
Polarity: positive

SentenceID: BHD#6:2
Polarity: positive

SentenceID: BHD#6:2
Polarity: positive

SentenceID: TM#5:0
Polarity: positive

SentenceID: TM#5:1
Polarity: positive

SentenceID: TM#5:2
Polarity: negative

SentenceID: TM#5:2
Polarity: positive

SentenceID: TM#5:3
Polarity: positive

SentenceID: TM#5:3
Polarity: positive

SentenceID: TM#5:4
Polarity: negative

SentenceID: TM#6:0
Polarity: positive

SentenceID: TM#6:1
Polarity: positive

SentenceID: TM#6:2
Polarity: positive

SentenceID: TM#6:2
Polarity: positive

SentenceID: TM#6:3
Polarity: negative

SentenceID: TM#6:4
Polarity: neutral

SentenceID: TM#6:5
Polarity: negative

SentenceID: TM#6:5
Polarity: positive

SentenceID: TM#6:6
Polarity: positive

SentenceID: TVU#6:0
Polarity: positive

SentenceID: TVU#6:2
Polarity: positive

SentenceID: TVU#6:3
Polarity: positive

SentenceID: TVU#6:3
Polarity: positive

SentenceID: TVU#6:4
Polarity: positive

SentenceID: TVU#6:4
Polarity: positive

SentenceID: TVU#6:5
Polarity: positive

SentenceID: TVU#6:5
Polarity: positive

SentenceID: TVU#6:6
Polarity: positive

SentenceID: TVU#6:7
Polarity: positive

SentenceID: TVU#6:8
Polarity: positive

SentenceID: TVU#6:8
Polarity: positive

SentenceID: TVU#6:9
Polarity: positive

SentenceID: ADLT#3:0
Polarity: positive

SentenceID: ADLT#3:1
Polarity: positive

SentenceID: ADLT#3:2
Polarity: positive

SentenceID: ADLT#3:3
Polarity: positive

SentenceID: ADLT#3:3
Polarity: positive

SentenceID: ADLT#3:3
Polarity: positive

SentenceID: TFS#5:2
Polarity: negative

SentenceID: TFS#5:4
Polarity: positive

SentenceID: TFS#5:5
Polarity: positive

SentenceID: TFS#5:6
Polarity: negative

SentenceID: TFS#5:7
Polarity: neutral

SentenceID: TFS#5:7
Polarity: neutral

SentenceID: TFS#5:7
Polarity: negative

SentenceID: TFS#5:8
Polarity: positive

SentenceID: TFS#5:8
Polarity: negative

SentenceID: TFS#5:12
Polarity: negative

SentenceID: TFS#5:13
Polarity: negative

SentenceID: TFS#5:16
Polarity: neutral

SentenceID: TFS#5:17
Polarity: positive

SentenceID: TFS#5:18
Polarity: negative

SentenceID: TFS#5:19
Polarity: negative

SentenceID: TFS#5:20
Polarity: negative

SentenceID: TFS#5:20
Polarity: negative

SentenceID: TFS#5:21
Polarity: negative

SentenceID: TFS#5:21
Polarity: negative

SentenceID: TFS#5:23
Polarity: neutral

SentenceID: TFS#5:23
Polarity: negative

SentenceID: TFS#5:24
Polarity: negative

SentenceID: TFS#5:24
Polarity: negative

SentenceID: TFS#5:25
Polarity: positive

SentenceID: TFS#5:25
Polarity: negative

SentenceID: TFS#5:26
Polarity: neutral

SentenceID: TFS#5:26
Polarity: neutral

SentenceID: TFS#5:27
Polarity: negative

SentenceID: TFS#5:29
Polarity: negative

SentenceID: TFS#5:30
Polarity: negative

SentenceID: TFS#5:30
Polarity: negative

SentenceID: CLF#3:0
Polarity: negative

SentenceID: CLF#3:0
Polarity: negative

SentenceID: CLF#3:1
Polarity: negative

SentenceID: CLF#3:3
Polarity: negative

SentenceID: CLF#3:3
Polarity: negative

SentenceID: CLF#3:4
Polarity: negative

SentenceID: CLF#3:4
Polarity: negative

SentenceID: CLF#3:6
Polarity: negative

SentenceID: CLF#3:6
Polarity: negative

SentenceID: CLF#3:7
Polarity: negative

SentenceID: CLF#3:7
Polarity: negative

SentenceID: CLF#3:8
Polarity: neutral

SentenceID: CLF#3:9
Polarity: negative

SentenceID: CLF#3:10
Polarity: positive

SentenceID: CLF#3:11
Polarity: positive

SentenceID: CLF#3:12
Polarity: positive

SentenceID: CLF#3:13
Polarity: negative

SentenceID: CLF#3:13
Polarity: negative

SentenceID: CLF#3:14
Polarity: negative

SentenceID: CLF#3:16
Polarity: negative

SentenceID: CLF#3:17
Polarity: negative

SentenceID: CLF#3:18
Polarity: negative

SentenceID: CLF#3:19
Polarity: negative

SentenceID: CLF#3:20
Polarity: negative

SentenceID: CLF#3:21
Polarity: negative

SentenceID: CLF#3:22
Polarity: negative

SentenceID: CLF#3:23
Polarity: negative

SentenceID: CLF#3:23
Polarity: negative

SentenceID: CLF#3:24
Polarity: negative

SentenceID: CLF#3:25
Polarity: negative

SentenceID: CLF#3:26
Polarity: negative

SentenceID: CLF#3:27
Polarity: negative

SentenceID: CLF#3:27
Polarity: negative

SentenceID: CLF#3:28
Polarity: negative

SentenceID: CLF#3:28
Polarity: negative

SentenceID: CLF#3:28
Polarity: negative

SentenceID: CLF#3:29
Polarity: negative

SentenceID: CLF#3:29
Polarity: negative

SentenceID: CLF#3:30
Polarity: negative

SentenceID: CLF#3:30
Polarity: negative

SentenceID: CLF#3:31
Polarity: negative

SentenceID: CLF#3:31
Polarity: negative

SentenceID: CLF#3:31
Polarity: negative

SentenceID: CLF#3:32
Polarity: negative

SentenceID: BFC#9:0
Polarity: negative

SentenceID: BFC#9:4
Polarity: positive

SentenceID: BFC#9:5
Polarity: neutral

SentenceID: BFC#9:6
Polarity: negative

SentenceID: BFC#9:7
Polarity: negative

SentenceID: BFC#9:8
Polarity: positive

SentenceID: BFC#9:8
Polarity: negative

SentenceID: BFC#9:8
Polarity: positive

SentenceID: BFC#9:9
Polarity: neutral

SentenceID: BFC#9:9
Polarity: negative

SentenceID: BFC#9:10
Polarity: negative

SentenceID: BFC#9:14
Polarity: negative

SentenceID: BFC#9:16
Polarity: neutral

SentenceID: BFC#9:17
Polarity: positive

SentenceID: BFC#9:18
Polarity: positive

SentenceID: BFC#9:18
Polarity: negative

SentenceID: BFC#9:18
Polarity: negative

SentenceID: BFC#9:19
Polarity: positive

SentenceID: BFC#9:20
Polarity: negative

SentenceID: BFC#9:21
Polarity: positive

SentenceID: BFC#9:21
Polarity: negative

SentenceID: BFC#9:22
Polarity: negative

SentenceID: BFC#9:23
Polarity: negative

SentenceID: BG#5:0
Polarity: positive

SentenceID: BG#5:2
Polarity: positive

SentenceID: BG#5:3
Polarity: neutral

SentenceID: BG#5:3
Polarity: positive

SentenceID: BG#5:4
Polarity: positive

SentenceID: BG#5:4
Polarity: negative

SentenceID: BG#5:4
Polarity: negative

SentenceID: BG#5:4
Polarity: negative

SentenceID: BG#5:5
Polarity: negative

SentenceID: BG#5:6
Polarity: negative

SentenceID: BG#5:6
Polarity: negative

SentenceID: BG#5:7
Polarity: positive

SentenceID: BG#5:7
Polarity: negative

SentenceID: BG#8:0
Polarity: positive

SentenceID: BG#8:1
Polarity: positive

SentenceID: BG#8:3
Polarity: negative

SentenceID: BG#8:3
Polarity: positive

SentenceID: BG#8:5
Polarity: positive

SentenceID: BG#8:5
Polarity: positive

SentenceID: BG#8:6
Polarity: positive

SentenceID: DBG#2:0
Polarity: negative

SentenceID: DBG#2:3
Polarity: negative

SentenceID: DBG#2:3
Polarity: negative

SentenceID: DBG#2:4
Polarity: negative

SentenceID: DBG#2:6
Polarity: negative

SentenceID: DBG#2:6
Polarity: negative

SentenceID: DBG#2:8
Polarity: negative

SentenceID: DBG#2:8
Polarity: negative

SentenceID: DBG#2:9
Polarity: negative

SentenceID: DBG#2:10
Polarity: negative

SentenceID: DBG#2:11
Polarity: negative

SentenceID: DBG#2:11
Polarity: negative

SentenceID: DBG#2:14
Polarity: negative

SentenceID: DBG#2:15
Polarity: negative

SentenceID: CLF#7:0
Polarity: positive

SentenceID: CLF#7:0
Polarity: positive

SentenceID: CLF#7:2
Polarity: positive

SentenceID: CLF#7:2
Polarity: positive

SentenceID: CLF#7:3
Polarity: positive

SentenceID: CLF#7:3
Polarity: positive

SentenceID: CLF#7:4
Polarity: positive

SentenceID: DBG#7:0
Polarity: negative

SentenceID: DBG#7:3
Polarity: negative

SentenceID: DBG#7:4
Polarity: neutral

SentenceID: DBG#7:4
Polarity: negative

SentenceID: DBG#7:5
Polarity: negative

SentenceID: DBG#7:6
Polarity: negative

SentenceID: DBG#7:7
Polarity: negative

SentenceID: DBG#7:8
Polarity: negative

SentenceID: DBG#7:9
Polarity: neutral

SentenceID: DBG#7:9
Polarity: neutral

SentenceID: DBG#7:10
Polarity: negative

SentenceID: DBG#7:12
Polarity: negative

SentenceID: DBG#7:14
Polarity: negative

SentenceID: DBG#7:15
Polarity: negative

SentenceID: DBG#7:16
Polarity: negative

SentenceID: ADLT#9:0
Polarity: positive

SentenceID: ADLT#9:2
Polarity: neutral

SentenceID: ADLT#9:3
Polarity: positive

SentenceID: ADLT#9:4
Polarity: positive

SentenceID: ADLT#9:5
Polarity: negative

SentenceID: ADLT#9:5
Polarity: positive

SentenceID: ADLT#9:6
Polarity: positive

SentenceID: ADLT#9:6
Polarity: positive

SentenceID: EVPK#4:0
Polarity: positive

SentenceID: EVPK#4:0
Polarity: positive

SentenceID: EVPK#4:1
Polarity: positive

SentenceID: EVPK#4:2
Polarity: neutral

SentenceID: EVPK#4:3
Polarity: positive

SentenceID: EVPK#4:3
Polarity: positive

SentenceID: EVPK#4:3
Polarity: positive

SentenceID: EVPK#4:4
Polarity: positive

SentenceID: EVPK#4:4
Polarity: positive

SentenceID: Z#6:0
Polarity: positive

SentenceID: Z#6:1
Polarity: positive

SentenceID: Z#6:1
Polarity: positive

SentenceID: Z#6:2
Polarity: neutral

SentenceID: Z#6:2
Polarity: positive

SentenceID: Z#6:2
Polarity: positive

SentenceID: Z#6:3
Polarity: positive

SentenceID: Z#6:3
Polarity: positive

SentenceID: Z#6:4
Polarity: positive

SentenceID: Z#6:4
Polarity: positive

SentenceID: Z#6:5
Polarity: positive

SentenceID: Z#6:5
Polarity: positive

SentenceID: Z#6:7
Polarity: positive

SentenceID: Z#6:9
Polarity: positive

SentenceID: Z#6:9
Polarity: positive

SentenceID: Z#6:11
Polarity: positive

SentenceID: Z#6:12
Polarity: neutral

SentenceID: Z#6:12
Polarity: positive

SentenceID: Z#6:13
Polarity: positive

SentenceID: Z#6:13
Polarity: positive

SentenceID: Z#6:14
Polarity: positive

SentenceID: Z#6:14
Polarity: positive

SentenceID: CLF#6:0
Polarity: negative

SentenceID: CLF#6:1
Polarity: negative

SentenceID: CLF#6:3
Polarity: negative

SentenceID: CLF#6:4
Polarity: negative

SentenceID: CLF#6:5
Polarity: negative

SentenceID: CLF#6:5
Polarity: negative

SentenceID: CLF#6:5
Polarity: negative

SentenceID: CLF#6:5
Polarity: negative

SentenceID: CLF#6:6
Polarity: negative

SentenceID: DBG#6:0
Polarity: positive

SentenceID: DBG#6:2
Polarity: positive

SentenceID: DBG#6:4
Polarity: neutral

SentenceID: DBG#6:5
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#6:7
Polarity: positive

SentenceID: DBG#1:0
Polarity: positive

SentenceID: DBG#1:0
Polarity: positive

SentenceID: DBG#1:0
Polarity: positive

SentenceID: BHD#1:0
Polarity: positive

SentenceID: BHD#1:0
Polarity: positive

SentenceID: BHD#1:0
Polarity: positive

SentenceID: TR#5:0
Polarity: neutral

SentenceID: TR#5:2
Polarity: positive

SentenceID: TR#5:3
Polarity: positive

SentenceID: TR#5:4
Polarity: positive

SentenceID: TR#5:6
Polarity: positive

SentenceID: TR#5:6
Polarity: negative

SentenceID: TR#5:6
Polarity: positive

SentenceID: Z#7:0
Polarity: negative

SentenceID: Z#7:1
Polarity: negative

SentenceID: Z#7:2
Polarity: negative

SentenceID: Z#7:2
Polarity: negative

SentenceID: Z#7:3
Polarity: negative

SentenceID: Z#7:3
Polarity: negative

SentenceID: Z#7:4
Polarity: negative

SentenceID: Z#7:5
Polarity: negative

SentenceID: Z#10:0
Polarity: positive

SentenceID: Z#10:2
Polarity: positive

SentenceID: Z#10:2
Polarity: negative

SentenceID: Z#10:3
Polarity: positive

SentenceID: Z#10:3
Polarity: positive

SentenceID: P#7:0
Polarity: positive

SentenceID: P#7:1
Polarity: positive

SentenceID: P#7:2
Polarity: positive

SentenceID: P#7:3
Polarity: positive

SentenceID: P#7:3
Polarity: neutral

SentenceID: P#7:4
Polarity: neutral

SentenceID: P#7:5
Polarity: positive

SentenceID: P#7:7
Polarity: positive

SentenceID: P#7:7
Polarity: positive

SentenceID: FF#4:0
Polarity: negative

SentenceID: FF#4:1
Polarity: negative

SentenceID: FF#4:2
Polarity: negative

SentenceID: FF#4:3
Polarity: negative

SentenceID: FF#4:4
Polarity: negative

SentenceID: FF#4:5
Polarity: negative

SentenceID: FF#4:6
Polarity: negative

SentenceID: FF#4:7
Polarity: negative

SentenceID: FF#4:7
Polarity: positive

SentenceID: FF#4:8
Polarity: positive

SentenceID: FF#4:8
Polarity: negative

SentenceID: FF#4:9
Polarity: positive

SentenceID: FF#4:9
Polarity: positive

SentenceID: FF#4:9
Polarity: positive

SentenceID: FF#4:9
Polarity: negative

SentenceID: FF#4:10
Polarity: negative

SentenceID: FF#4:10
Polarity: negative

SentenceID: FF#4:10
Polarity: negative

SentenceID: FF#4:11
Polarity: negative

SentenceID: FF#3:0
Polarity: negative

SentenceID: FF#3:1
Polarity: negative

SentenceID: FF#3:3
Polarity: positive

SentenceID: FF#3:5
Polarity: positive

SentenceID: FF#3:5
Polarity: negative

SentenceID: FF#3:6
Polarity: negative

SentenceID: FF#3:7
Polarity: negative

SentenceID: FF#3:7
Polarity: negative

SentenceID: FF#3:9
Polarity: negative

SentenceID: FF#3:9
Polarity: negative

SentenceID: FF#3:10
Polarity: negative

SentenceID: FF#3:11
Polarity: negative

SentenceID: FF#3:12
Polarity: negative

SentenceID: FF#3:14
Polarity: negative

SentenceID: FF#3:15
Polarity: negative

SentenceID: FF#3:16
Polarity: negative

SentenceID: FF#3:17
Polarity: negative

SentenceID: FF#3:19
Polarity: negative

SentenceID: FF#3:20
Polarity: negative

SentenceID: FF#3:22
Polarity: negative

SentenceID: FF#3:23
Polarity: negative

SentenceID: FF#3:23
Polarity: positive

SentenceID: FF#3:23
Polarity: positive

SentenceID: FF#3:24
Polarity: negative

SentenceID: FF#3:25
Polarity: negative

SentenceID: EVPK#9:0
Polarity: positive

SentenceID: EVPK#9:2
Polarity: positive

SentenceID: EVPK#9:3
Polarity: positive

SentenceID: EVPK#9:3
Polarity: positive

SentenceID: EVPK#9:4
Polarity: positive

SentenceID: CLF#4:0
Polarity: negative

SentenceID: CLF#4:1
Polarity: positive

SentenceID: CLF#4:1
Polarity: positive

SentenceID: CLF#4:1
Polarity: positive

SentenceID: CLF#4:1
Polarity: positive

SentenceID: CLF#4:3
Polarity: negative

SentenceID: CLF#4:5
Polarity: negative

SentenceID: CLF#4:6
Polarity: negative

SentenceID: CLF#4:10
Polarity: negative

SentenceID: CLF#4:11
Polarity: positive

SentenceID: CLF#4:12
Polarity: neutral

SentenceID: CLF#4:12
Polarity: negative

SentenceID: CLF#4:12
Polarity: negative

SentenceID: CLF#4:12
Polarity: negative

SentenceID: CLF#4:13
Polarity: negative

SentenceID: BHD#4:0
Polarity: positive

SentenceID: BHD#4:2
Polarity: positive

SentenceID: BHD#4:2
Polarity: neutral

SentenceID: BHD#4:3
Polarity: positive

SentenceID: BHD#4:3
Polarity: positive

SentenceID: BHD#4:4
Polarity: positive

SentenceID: BHD#4:4
Polarity: positive

SentenceID: BHD#4:5
Polarity: negative

SentenceID: BHD#4:6
Polarity: positive

SentenceID: BHD#4:6
Polarity: positive

SentenceID: BFC#7:0
Polarity: positive

SentenceID: BFC#7:0
Polarity: negative

SentenceID: BFC#7:2
Polarity: negative

SentenceID: BFC#7:3
Polarity: positive

SentenceID: BFC#7:3
Polarity: negative

SentenceID: BFC#7:3
Polarity: positive

SentenceID: BFC#7:4
Polarity: negative

SentenceID: BFC#7:4
Polarity: negative

SentenceID: BFC#7:4
Polarity: negative

SentenceID: BFC#7:5
Polarity: negative

SentenceID: BFC#7:6
Polarity: negative

SentenceID: BFC#7:7
Polarity: negative

SentenceID: BFC#7:8
Polarity: negative

SentenceID: BFC#7:9
Polarity: negative

SentenceID: BFC#7:10
Polarity: negative

SentenceID: BFC#7:11
Polarity: negative

SentenceID: BFC#7:11
Polarity: negative

SentenceID: BFC#7:12
Polarity: negative

SentenceID: ADLT#6:0
Polarity: negative

SentenceID: ADLT#6:2
Polarity: positive

SentenceID: ADLT#6:2
Polarity: positive

SentenceID: ADLT#6:2
Polarity: negative

SentenceID: ADLT#6:3
Polarity: positive

SentenceID: ADLT#6:4
Polarity: negative

SentenceID: ADLT#6:5
Polarity: negative

SentenceID: ADLT#6:6
Polarity: negative

SentenceID: ADLT#6:7
Polarity: negative

SentenceID: NP#9:0
Polarity: positive

SentenceID: NP#9:1
Polarity: positive

SentenceID: NP#9:1
Polarity: positive

SentenceID: NP#9:2
Polarity: positive

SentenceID: NP#9:3
Polarity: positive

SentenceID: TFS#2:0
Polarity: positive

SentenceID: TFS#2:1
Polarity: positive

SentenceID: TFS#2:1
Polarity: positive

SentenceID: TFS#2:2
Polarity: positive

SentenceID: TFS#2:3
Polarity: positive

SentenceID: BHD#8:0
Polarity: positive

SentenceID: BHD#8:1
Polarity: negative

SentenceID: BHD#8:1
Polarity: positive

SentenceID: BHD#8:2
Polarity: positive

SentenceID: BHD#8:2
Polarity: positive

SentenceID: P#3:0
Polarity: positive

SentenceID: P#3:2
Polarity: positive

SentenceID: P#3:2
Polarity: neutral

SentenceID: P#3:3
Polarity: positive

SentenceID: P#3:3
Polarity: positive

SentenceID: P#3:3
Polarity: negative

SentenceID: CLF#8:0
Polarity: negative

SentenceID: CLF#8:0
Polarity: negative

SentenceID: CLF#8:1
Polarity: negative

SentenceID: CLF#8:2
Polarity: negative

SentenceID: CLF#8:3
Polarity: negative

SentenceID: CLF#8:4
Polarity: negative

SentenceID: CLF#8:5
Polarity: negative

SentenceID: CLF#8:6
Polarity: negative

SentenceID: CLF#8:7
Polarity: negative

SentenceID: CLF#8:8
Polarity: negative

SentenceID: CLF#8:9
Polarity: negative

SentenceID: CLF#8:9
Polarity: negative

SentenceID: ADLT#11:0
Polarity: negative

SentenceID: ADLT#11:1
Polarity: neutral

SentenceID: ADLT#11:2
Polarity: negative

SentenceID: ADLT#11:2
Polarity: negative

SentenceID: ADLT#11:2
Polarity: negative

SentenceID: ADLT#11:3
Polarity: negative

SentenceID: ADLT#11:4
Polarity: negative

SentenceID: WE#1:0
Polarity: negative

SentenceID: WE#1:3
Polarity: negative

SentenceID: WE#1:3
Polarity: negative

SentenceID: WE#1:3
Polarity: negative

SentenceID: WE#1:4
Polarity: positive

SentenceID: WE#1:4
Polarity: negative

SentenceID: WE#1:4
Polarity: negative

SentenceID: FF#6:0
Polarity: negative

SentenceID: FF#6:3
Polarity: negative

SentenceID: FF#6:5
Polarity: negative

SentenceID: FF#6:5
Polarity: negative

SentenceID: FF#6:6
Polarity: negative

SentenceID: FF#6:7
Polarity: negative

SentenceID: FF#6:8
Polarity: negative

SentenceID: FF#6:10
Polarity: negative

SentenceID: FF#6:11
Polarity: negative

SentenceID: FF#6:12
Polarity: negative

SentenceID: FF#6:12
Polarity: negative

SentenceID: FF#10:0
Polarity: negative

SentenceID: FF#10:1
Polarity: positive

SentenceID: FF#10:2
Polarity: positive

SentenceID: FF#10:3
Polarity: positive

SentenceID: FF#10:4
Polarity: negative

SentenceID: FF#10:5
Polarity: negative

SentenceID: FF#10:5
Polarity: negative

SentenceID: FF#10:6
Polarity: negative

SentenceID: FF#10:7
Polarity: negative

SentenceID: FF#10:8
Polarity: negative

SentenceID: FF#10:9
Polarity: negative

SentenceID: FF#10:10
Polarity: negative

SentenceID: FF#10:11
Polarity: negative

SentenceID: FF#10:11
Polarity: negative

